pybind_interface/pybind_main.cpp

```diff
@@ -48,20 +48,12 @@ T parseOptions(const py::dict &options, const char *key) {
   return value.cast<T>();
 }
 
-Circuit<GateQSim<float>> getCircuit(const py::dict &options) {
-  Circuit<GateQSim<float>> circuit;
-  std::string circuit_str;
+Circuit<Cirq::GateCirq<float>> getCircuit(const py::dict &options) {
   try {
-    circuit_str = parseOptions<std::string>(options, "c\0");
+    return options["c\0"].cast<Circuit<Cirq::GateCirq<float>>>();
   } catch (const std::invalid_argument &exp) {
     throw;
   }
-  std::stringstream ss(circuit_str);
-  if (!CircuitQsimParser<IO>::FromStream(std::numeric_limits<unsigned>::max(),
```

Comment:
`#include "../lib/circuit_qsim_parser.h"` above should be deleted.

Reply:
Done.